examples/plugin/Hello_world/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+
+find_package(ignition-cmake2 REQUIRED)
+
+project(Hello_world)
+
+find_package(ignition-plugin1 REQUIRED COMPONENTS register)
```

Comment:
Could we use `ign_find_package` command from ign-cmake here and for ignition-gazebo4 below? IT does not make any difference right now but using the ign-cmake wrapper can provide extra benefits in the future.

Reply:
@j-rivero, `find_package` in line 7?

Reply:
> @j-rivero, `find_package` in line 7?

I have made the changes.